### ISSUE TITLE
Twitch Live-Chat Cookie-Banner not scrollable

### ISSUE THREAD

**Issue by xephos1one** [futo-org/grayjay-android#784]
The Cookie-Banner in the Live-Chat is not scrollable and therefore can't be closed.
You can't accept nor decline the use of Cookies so you can't read the Chat.

Android 14
App-Version: 233
Twitch-Plugin-Version: 14

![Screenshot_20240207-213342](https://github.com/futo-org/grayjay-android/assets/39647898/63173de9-852c-41dc-819d-08e87b29c7fe)

**Comment by spieler1x** [comment]
I have the same issue on both version 233 and the lastest 235. So far Grayjay works really Good, but you guys need to fully focus on the Twitch plugin, cause this is poorly made.


Android 13
Version: 233 & 235
Twitch Plugin: 14

![Screenshot_20240220_184850_Grayjay](https://github.com/futo-org/grayjay-android/assets/160286417/2aa897c4-d84a-4b52-a8c0-ca8cc022f343)

**Comment by xephos1one** [comment]
> @xephos1one is the issue still present? Think we released a fix for this some time ago...

The cookie banner currently is coming up but goes away a second later.
But the buttons are now clickable even in the second it's there - at least for me, I have a 21:9 display and don't know if they are there with 16:9.
Do you mean it's fixed because the cookie banner comes up but goes away shortly after?
Is it possible to completely get rid of the banner or make the banner scrollable and don't hide it so users could accept or personalize cookie settings themselves?

Here's a screenshot of the current cookie banner while it was there:
![Screenshot_20250106-220655](https://github.com/user-attachments/assets/153391d1-57f7-4829-a5c9-124c74a9ff46)

**Comment by Zvonimir-FUTO** [comment]
Yeah it disappearing after a second was a quick hack to fix it. Will look into improving this but it is not high priority right now.
